.github/workflows/build_docker.yml

```diff
@@ -5,9 +5,6 @@ on:
       - main
       - dev
   pull_request:
-    branches:
```

Comment:
Is removing branch spec here deliberate?

Reply:
Yeah. I see no reason to skip these tests for any PR. In the event we get a long-lived feature branch, this test should probably still run.